### PR TITLE
fix(ci): only set vx.y.z to latest release

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue #

### CODE REVIEW

**Review by innobead — DISMISSED** [review]
LGTM!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -40,6 +40,8 @@ jobs:
       uses: softprops/action-gh-release@v2
       if: startsWith(github.ref, 'refs/tags/')
       with:
+        prerelease: ${{ !matches(github.ref, '^refs/tags/v[0-9]+\\.[0-9]+\\.[0-9]+$') }}
+        draft: true
```
Comment (innobead) [inline .github/workflows/build.yml:44]: `draft` can be true when it's GA release.

Comment (derekbit) [reply .github/workflows/build.yml:44]: How about setting it t to `false` for all version by default as the `longhorn/longhorn` release?

Comment (innobead) [reply .github/workflows/build.yml:44]: Sounds good. Missed that part!

Comment (innobead) [reply .github/workflows/build.yml:44]: Let's make it simple.

Make draft: true always. When making the release of Longhorn repo as latest (ga), then we just update the recent draft release of cli repo to latest. WDYT?

Comment (derekbit) [reply .github/workflows/build.yml:44]: Sounds good!

**Review by innobead — APPROVED** [review]

**Comment by derekbit** [comment]
@mergify backport v1.7.x

**Pull request merged**